### PR TITLE
DHFPROD-7701: Update to DH 5.5.2

### PR DESCRIPTION
**Pull request by ryanjdew**
This updates DHCCE to use DH 5.5.2. 

The tests would occasionally timeout for me on the UploadServiceTest, so I'd like to what CircleCI does.

### CODE REVIEW

**Review by Sanjeevani19 — COMMENTED** [review]
@ryanjdew Looks like 14 middle tier tests are still failing https://app.circleci.com/pipelines/github/marklogic-community/data-hub-central-community/231/workflows/896569fc-db1b-4e03-9ac6-97bc17d52364/jobs/457, Please look into them

**Review by Sanjeevani19 — COMMENTED** [review]
@ryanjdew Around 134 tests failed on most recent changes, https://app.circleci.com/pipelines/github/marklogic-community/data-hub-central-community/233/workflows/c4540727-71f3-4efd-a677-91b05f79aa10/jobs/462, please look into them, Looks unauthorized error on all of them, might be a permission issue

**Review by Sanjeevani19 — APPROVED** [review]

File: .circleci/run-tests.sh [hunk 1]
```diff
@@ -9,14 +9,24 @@ mkdir conceptConnectorModels
 
 echo "plugins {
 	id 'net.saliman.properties' version '1.4.6'
-	id 'com.marklogic.ml-data-hub' version '5.2.5'
+	id 'com.marklogic.ml-data-hub' version '5.5.2'
```
Comment (Sanjeevani19) [inline .circleci/run-tests.sh:12]: @ryanjdew Should this still be pointing to 5.5.2 ?

**Review by Sanjeevani19 — APPROVED** [review]
@ryanjdew confirmed that both UI and middle-tier tests passed after updating to latest 5.5 snapshot(5.5.3), tests would fail in the pipeline since circle-ci is still pointing to 5.5.2, Once 5.5.3 is released need to update  .circleci/run-tests.sh and build.gradle files with 5.5.3 so the tests would run https://project.marklogic.com/jira/browse/DHFPROD-7816. Approving the PR for now.

**Review by Sanjeevani19 — COMMENTED** [review]
@ryanjdew One UI test is failing on latest build https://app.circleci.com/pipelines/github/marklogic-community/data-hub-central-community/236/workflows/39a98691-07f6-4da9-863e-692c0c99d499/jobs/469, Please look into it

**Review by Sanjeevani19 — APPROVED** [review]
@ryanjdew rerunning the tests fixed the UI test failure, might be intermittent. Looks good, approving the PR

**Review by xmlnovelist — APPROVED** [review]
This update aligns with data hub 5.5.3

**Pull request merged**